tests/test_resolvers.py

```diff
@@ -7,6 +21,12 @@
     ResolutionImpossible,
     Resolver,
 )
+from resolvelib.resolvers import Resolution  # type: ignore
```

Comment:
What errors does this ignore?

Reply:
`Resolution` is not part of the public interface declared in `src/resolvelib/resolvers.pyi`. My reasoning was that for testing that does not matter. I meant to ask about this but I forgot about it. The error is as follows:
```
tests/test_resolvers.py:24: error: Module "resolvelib.resolvers" has no attribute "Resolution"; maybe "ResolutionError"?
Found 1 error in 1 file (checked 1 source file)
```

Reply:
We can add `Resolution` to the type stubs; I think someone would need it at some point anyway.

Reply:
All right, I'll do that, thanks.

Reply:
Done. I did have to add another more specific type ignore because the test case monkeypatches a private method. I think that should be acceptable?